docs/samples/Microsoft.ML.Samples/Dynamic/TensorFlow/TextClassification.cs

```diff
@@ -43,49 +51,68 @@ public static void Example()
                );
 
             // Load the TensorFlow model once.
-            //      - Use it for quering the schema for input and output in the model
+            //      - Use it for quering the schema for input and output in the
+            //            model
             //      - Use it for prediction in the pipeline.
-            var tensorFlowModel = mlContext.Model.LoadTensorFlowModel(modelLocation);
+            var tensorFlowModel = mlContext.Model.LoadTensorFlowModel
```

Comment:
Not sure if it should be
```
LoadTensorFlowModel(
    modelLocation)
```

Reply:
Yep, changed that on the most recent commit.